articles/virtual-desktop/troubleshoot-vm-configuration.md

```diff
@@ -337,6 +337,10 @@ If your users see an error that says, “We couldn't connect to the remote PC be
 
 To learn more about this policy, see [Allow log on through Remote Desktop Services](/windows/security/threat-protection/security-policy-settings/allow-log-on-through-remote-desktop-services).
 
+## Issues with golden image
+
+Having the Azure Virtual Desktop agent as part of the golden image is not supported. The agent must always be installed once the golden image is being deployed and not as part of the golden image itself.
```

Comment:
```suggestion
Base images don't currently include or support the Azure Virtual Desktop agent. You can only install the agent after you deploy the base image.
```

Reply:
Hi Heidi, the golden image is different from the base images from Azure marketplace. A golden image is a custom image IT admins create that is used for their AVD deployments.